src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/HoverTests.cs

```diff
@@ -44,4 +44,22 @@ public async Task Hover_OverComponentAttribute()
         const string ExpectedResult = "string? SurveyPrompt.Title { get; set; }";
         Assert.Equal(ExpectedResult, hoverString);
     }
+
+    [IdeFact]
+    public async Task Hover_OverHTMLWithCatchAll()
+    {
+        // Open the file
+        await TestServices.SolutionExplorer.OpenFileAsync(RazorProjectConstants.BlazorProjectName, RazorProjectConstants.CounterRazorFile, ControlledHangMitigatingCancellationToken);
+
+        await TestServices.Editor.SetTextAsync("""
+            <button @onclick="@OnClick">Click me</button>
+            """, HangMitigatingCancellationToken);
+
+        // Act
+        var hoverString = await TestServices.Editor.GetHoverStringAsync(3, ControlledHangMitigatingCancellationToken);
+
+        // Assert
+        const string ExpectedResult = "The button element represents a button labeled by it's contents.";
```

Comment:
I couldn't run this locally due to test harness setup failures, so I'm just kind of hoping this is right based on manually testing